region/brinstar/blue/Blue Brinstar Boulder Room.json

```diff
@@ -182,6 +215,79 @@
         "Emphasis on Careful.  There is no jump."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Leave With Mockball",
+      "requires": [],
+      "exitCondition": {
+        "leaveWithMockball": {
+          "remoteRunway": {
+            "length": 16,
+            "openEnd": 1
+          },
+          "landingRunway": {
+            "length": 1,
+            "openEnd": 1
+          },
+          "minExtraRunSpeed": "$1.2",
+          "maxExtraRunSpeed": "$3.9"
+        }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}]
+    },
+    {
+      "link": [1, 2],
+      "name": "Leave With Spring Ball Bounce",
+      "requires": [],
+      "exitCondition": {
+        "leaveWithSpringBallBounce": {
+          "remoteRunway": {
+            "length": 16,
+            "openEnd": 1
+          },
+          "landingRunway": {
+            "length": 1,
+            "openEnd": 1
+          },
+          "minExtraRunSpeed": "$1.2",
+          "maxExtraRunSpeed": "$3.9",
+          "movementType": "uncontrolled"
+        }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}]
+    },
+    {
+      "link": [1, 2],
+      "name": "Leave Space Jumping",
+      "requires": [],
```

Comment:
```suggestion
      "requires": [
        "canPreciseSpaceJump"
      ],
```

Reply:
It could depend on how much runway you use and the kind of short hop you're going for if its a trickyJump or canPreciseSpaceJump